test/CppTest_UWP/pch.h

```diff
@@ -0,0 +1,20 @@
+﻿#pragma once
+#include <windows.h>
+#include <unknwn.h>
+#include <wil/cppwinrt.h>
+#include <wil/resource.h>
+#include <wil/result_macros.h>
+#include <restrictederrorinfo.h>
+#include <hstring.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Windows.ApplicationModel.Activation.h>
+#include <winrt/Windows.UI.Xaml.h>
```

Comment:
Needing all of these huge Xaml namespaces just for testing is unfortunate as its going to make rebuilding the tests take a little while.

Reply:
Many of the tests here are going to require some amount of UX. Yes, it's unfortunate. Any suggestions? I'm just going off the existing "UWP unit tests" projects.

Reply:
I would just add the specific namespaces as you need them when you need them. Other than that, there's not much you can do about Windows.UI's heftiness until C++ modules finally arrive.